lib/app/presentation/utils/app_colors.dart

```diff
@@ -0,0 +1,6 @@
+import 'package:flutter/material.dart';
+
+class AppColors {
+  static const Color primaryOrange = Color(0xFFFF7622);
```

Comment:
Let us change the color strategy to use primary, secondary, tertiary, and accent colors only then predefinced colors for other to avoid clogging and duplicating other colors